packages/mdc-ripple/foundation.js

```diff
@@ -471,7 +471,7 @@ export default class MDCRippleFoundation extends MDCFoundation {
     this.frame_ = this.adapter_.computeBoundingRect();
 
     const maxDim = Math.max(this.frame_.height, this.frame_.width);
-    const surfaceDiameter = Math.sqrt(Math.pow(this.frame_.width, 2) + Math.pow(this.frame_.height, 2));
+    const surfaceDiameter = this.frame_.width;
```

Comment:
I suggest we change this to `Math.min(this.frame_.width, this.frame_.height)` so when unbounded ripple is applied to non-square surface, it will always start within the container.